pages/community/ambassador.js

```diff
@@ -0,0 +1,210 @@
+import GenericLayout from "../../components/layout/GenericLayout";
+import ambassadorsList from "../../config/TSC_MEMBERS.json";
+import TwitterSVG from "../../components/icons/twitter_2";
+import GithubSVG from "../../components/icons/github_2";
+import LinkedInSVG from "../../components/icons/linkedin_2";
+import {sortBy} from 'lodash';
+
+function addAdditionalUserInfo(user) {
+  const userData = {
+    ...user,
+  };
+
+  // if username is not present, use the github username
+  if (!userData.name) userData.name = userData.github;
+
+  // add social links
+  if (userData.github)
+    userData.github = `https://www.github.com/${userData.github}`;
+  if (userData.linkedin)
+    userData.linkedin = `https://www.linkedin.com/in/${userData.linkedin}`;
+  if (userData.twitter)
+    userData.twitter = `https://www.twitter.com/${userData.twitter}`;
+
+  // add avatar url
+  // github redirects to avatar url using `https://www.github.com/<username>.png`
+  userData.avatarUrl = userData.github + ".png";
+
+  // make repo links
+  userData.repos = userData.repos.map((repoName) => ({
+    name: repoName,
+    url: `https://www.github.com/asyncapi/${repoName}`,
+  }));
+
+  return userData;
+}
+
+export default function ambassadors() {
+  const description =
+    "Meet the current AsyncAPI ambassadors and learn how you can become one.";
+  const image = "/img/social/website-card.jpg";
+
+  const ambassadors = sortBy(ambassadorsList.map((user) => addAdditionalUserInfo(user)),['name']);
+
+  return (
+    <GenericLayout
+      title="AsyncAPI ambassadors"
+      description={description}
+      image={image}
+      wide
+    >
+      <div className="py-12 relative max-w-xl mx-auto px-4 sm:px-6 lg:px-8 lg:max-w-screen-xl">
+        <div className="grid lg:grid-cols-3 lg:gap-8">
+          <div>
+            <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            Who is an ambassador?
+            </h3>
+            <p className="my-4 text-base text-gray-500 lg:text-center">
+            The AsyncAPI Ambassador brings AsyncAPI closer to the users and potential users of the project. Provide users and community members with the necessary tools and resources:
+They promote cooperation between users and disseminate the necessary tools to any specification user.
+They are engaged with the project in some way, either as a contributor, blogger, speaker, etc.
+They should be willing to speak at community events or write technical content, such as blog posts, solving community needs.
+            </p>
+          </div>
+          <div>
+            <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            How to become an ambassador?
+            </h3>
+            <p className="my-4 text-base text-gray-500 lg:text-center">
+            To be an official AsyncAPI ambassador, there is no more important requirement than having the desire to work and spread the AsyncAPI tools.
```

Comment:
"To be an official AsyncAPI ambassador, **there is no more important requirement than having the desire to work and spread the AsyncAPI tools**."

What does that mean though, really? as a user... if I landed on this page and wanted to understand the steps to become an ambassador, this sentence would only confuse me. It does not really tell me what work I have to actually do. It also alludes to helping spread something about the tools, but that isn't clear. (typo in the text aside)

@derberg @Barbanio  what are actual items or steps a community member can do to become eligible to become an ambassador?

Reply:
The idea is that the model currently in place will be used to add members to the Technical Steering Committee through PR. And later, the TSC or its delegated community repository maintainers will evaluate the applications according to the fulfillment of the requirements. It will be done through a PR flow.